components/activity_log_modal/activity_log_modal.test.tsx

```diff
@@ -1,13 +1,12 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
+import React, { MouseEvent } from 'react';
```

Comment:
You're using `React.MouseEvent` below, which is fine, so we can remove this import
```suggestion
import React from 'react';
```